terraspace/app/stacks/peer/tfvars/base.tfvars

```diff
@@ -0,0 +1,7 @@
+vpc = {
+  name = "<%= expansion(':ENV') %>-ep-peer"
+}
+cluster_version          = 1.21
```

Comment:
shouldn't this be a string?

Reply:
Not going to do that because VPC object is going to have new attributes based on another comments (cidr, private_subnets, etc...)